.gitmodules

```diff
@@ -4,12 +4,12 @@
   branch = dev/emc
 [submodule "ccpp/framework"]
   path = ccpp/framework
-  url = https://github.com/NCAR/ccpp-framework
-  branch = main
+  url = https://github.com/peverwhee/ccpp-framework
```

Comment:
This needs to be reverted.

Reply:
done.

Reply:
@dustinswales Why? We want this to point to the PR branch of ccpp-framework for testing, do we not?

Reply:
Doh. face palm.